hammer_cli_reference/playbook.yaml

```diff
@@ -0,0 +1,100 @@
+---
+- name: "Check prerequisites on developer-system"
+  connection: local
+  hosts: developer-system
+  vars:
+    repo_path: "/home/user"
+    repo_remote: "git@github.com:theforeman/foreman-documentation.git"
+
+  tasks:
+    - name: "Verify that user input is overwritten"
+      ansible.builtin.assert:
+        that: repo_path != "/home/user"
+
+    - name: "Verify that shellscript is present"
+      ansible.builtin.stat:
+        path: "{{ repo_path }}/guides/scripts/generate-hammer-reference.sh"
+      register: result
+
+    - name: "End playbook if shellscript is not present"
+      ansible.builtin.fail:
+        msg: "Shellscript to convert markdown to asciidoc is not present or executable"
+      when:
+        - not result.stat.executable
+        - not result.stat.exists
+
+    - name: "Delete markdown file if present"
+      ansible.builtin.file:
+        path: "{{ repo_path }}/guides/hammer_full_help.md"
+        state: absent
+
+    - name: "Verify that foreman-documentation is present"
+      ansible.builtin.git:
+        clone: false
+        dest: "{{ repo_path }}"
+        repo: "{{ repo_remote }}"
+        update: false  # noqa: latest
+        version: "HEAD"
+
+- name: "Generate Hammer CLI help"
+  hosts: foreman-katello
+
+  tasks:
+    - name: "Update DNF cache"
+      ansible.builtin.dnf:
+        update_cache: true
+
+    - name: "Install all available Hammer CLI packages"
+      ansible.builtin.package:
+        name: "rubygem-hammer_cli*"
```

Comment:
I'm not sure if installation of these rubygems is sufficient. In the Hammer guide, [we have a different set of steps](https://docs.theforeman.org/nightly/Hammer_CLI/index-katello.html#installing-standalone-hammer).

Reply:
The linked guide is about installer Hammer CLI on a standalone host, aka. a host that's not Foreman+Katello server or a Smart Proxy server.